src/com/itacademy/eshop/product/Product.java

```diff
@@ -33,6 +37,17 @@ public ArrayList<Review> getReviews() {
         return reviews;
     }
 
+    public void setDescription(String productDescription) {
+        description.setProductDescription(productDescription);
+        description.setDescriptionAdditionTime();
+    }
+
+    public Description getDescription() {return description;}
+
+    public String appendDescription(Product product) {
```

Comment:
you method indicates that you will append a description but for me it looks like that you want to return description and the time. you could achieve similar functionality if you override toString and print the way you want to print these values. then you would just need to get the description and then print it out using toString()